nornir_netbox/plugins/inventory/netbox.py

```diff
@@ -211,13 +214,19 @@ def __init__(
         self.filter_parameters = filter_parameters
         self.include_vms = include_vms
         self.use_platform_slug = use_platform_slug
+        self.use_napalm = use_napalm
 
         self.session = requests.Session()
         self.session.headers.update({"Authorization": f"Token {nb_token}"})
         self.session.verify = ssl_verify
 
     def load(self) -> Inventory:
 
+        platforms: List[Dict[str, Any]] = []
```

Comment:
We should fence this with a `if use_napalm`. This shouldn't be called if there is no intention for that.

Reply:
Good idea, updated